backend/src/Controller/ChatRoom/ChatRoomController.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace App\Controller\ChatRoom;
+
+use App\Controller\BaseController;
+use App\Service\ChatRoom\ChatRoomService;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Serializer\SerializerInterface;
+use OpenApi\Annotations as OA;
+use Nelmio\ApiDocBundle\Annotation\Security;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+
+/**
+ * fetch chat room between admin and user.
+ * @Route("v1/chatroom/")
+ */
+class ChatRoomController extends BaseController
+{
+    private ChatRoomService $chatRoomService;
+
+    public function __construct(SerializerInterface $serializer, ChatRoomService $chatRoomService)
+    {
+        parent::__construct($serializer);
+        $this->chatRoomService = $chatRoomService;
+    }
+
+    /**
+     * store and captain :fetch chat room for user
+     * @Route("chatroom", name="getChatRoom", methods={"GET"})
```

Comment:
How about granting the API for the required users?

Reply:
It is used for any user who owns a chat room, so there is no need to specify the type of user, this router is not currently used, it is only available for the time of need